### ISSUE TITLE
GeoIP localhost

### ISSUE THREAD

**Issue by idezigns** [antonioribeiro/tracker#443]
I installed the tracker on my Laravel 5.8 project, the only thing I cannot get to populate is the GeoIP, I keep getting null, and no database entries.

I have 

log_geoip' => true

'geoip_database_path' set to storage_path('app/geoip'), 
       --database is in location; storage/app/geoip with the following files;

GeoLite2-City.md5
GeoLite2-City.mmdb
GeoLite2-City.mmdb.gz

I also have;

'do_not_track_ips' => [
       // '127.0.0.0/24', /// range 127.0.0.1 - 127.0.0.255
    ],

The only thing I considered is maybe because I am testing it locally via Laragon, that it is ignoring the 127.0.0.1 result.
I would like to confirm it works however, and I do not know how to go about this.

**Comment by idezigns** [comment]
Also, where is the country database?

**Comment by jimcouts** [comment]
You need another computer on the network to view you on your WAN, it's best to let LAN view that resolves off your network open it up on your router to allow someone externally (briefly) to come and view your site test site to see if it gets rid of the null. Did you happen to look in the table to see if it's even populated anything?

![geoip](https://user-images.githubusercontent.com/8333999/58145635-b2545f80-7c07-11e9-93b9-5a169a544c46.png)

**Comment by idezigns** [comment]
Yes, I've ran my phone on the network, my macbook pro on the network, and even opened ports to run my phone under phone carrier to log in, and nothing
